prow/scripts/resources/README.md

```diff
@@ -0,0 +1,22 @@
+## Overview
+
+The folder contains files that are directly used by Prow pipeline scripts.
+
+## Directory structure
+
+```
+├── kube-dns-stub-domains-patch.yaml                          # Enable stubdomain build.kyma-project.io and provides Google root DNS servers IPs.
+├── limitrange-patch.yaml                                     # Increases the kyma-system Namespace maximum memory request for containers.
+├── prometheus-operator-stackdriver-patch.yaml                # Injects the Stackdriver collector sidecar, sets metric filters, and enables scraping Stackdriver target.
+└── prometheus-operator-additional-scrape-config.yaml         # Additionall scrape configuration for Prometheus operator.
+├── kube-dns-stub-domains-patch.yaml                          # Enables stubdomain build.kyma-project.io and provides Google root DNS servers IPs.
```

Comment:
From this point, the structure is repeated. We can delete it.

Reply:
done